memiavl/import.go

```diff
@@ -0,0 +1,93 @@
+package memiavl
+
+import (
+	"math"
+
+	"github.com/cosmos/iavl"
+	"github.com/pkg/errors"
+)
+
+// Import a stream of `iavl.ExportNode`s into a new snapshot.
+func Import(dir string, version int64, nodes chan *iavl.ExportNode, writeHashIndex bool) (returnErr error) {
```

Comment:
limit to single direction to avoid nodes <- nil
```suggestion
func Import(dir string, version int64, nodes <-chan *iavl.ExportNode, writeHashIndex bool) (returnErr error) {
```